### DISCUSSION TITLE
Battery or no battery ?

### DISCUSSION THREAD

**Discussion by rofo69** [BigBobbas/ESP32-S3-Box3-Custom-ESPHome#60] [General]
I'm using an ESP32-Box3B, which is the white variant without some 'accessories', although I am not sure what these are.
What I'm curious about is whether this has an internal battery, as I've seen videos of the blue variant used wirelessly, and Robs firmware clearly has a battery indicator on it.

However, I've had the thing plugged in for days, and the battery state never changes (no charge). I've tried via the dock, and the usb c port on the side of the screen, no difference... Also tried different chargers, no difference.

**Comment by BigBobbas** [comment]
No there is no battery, the battery is in the version with the blue base and the battery fits in the base. 
There is a 'no sensor' version of the config, which is for the version like yours without a sensor basee, which also has the battery.

Cheers
Rob

**Comment by rofo69** [comment]
Thanks for the update. I'll redo the firmware on mine to match the no config version.
